src/utils/openCloseComponent.ts

```diff
@@ -60,6 +61,26 @@ function transitionEnd(event: TransitionEvent): void {
     this.open ? this.onOpen() : this.onClose();
   }
 }
+
+/**
+ * Helper to determine globally set transition duration setting.
+ *
+ * @param component
+ */
+export function onToggleUnanimatedComponent(component: OpenCloseComponent): void {
```

Comment:
Is there another name we could use here? I'm not sure what 'unanimated component' means in the context of open/close. 😅